cadasta/common/setting.py

```diff
@@ -149,3 +149,26 @@ def get_path_data(organization_slug=None, project_slug=None):
             '%s.geojson' % project_slug
         )
     return data_path
+
+
+def get_path_database(database=None):
+    """ Path databased based on database.
```

Comment:
No space in the begining of the docstring. See : https://github.com/inasafe/inasafe/wiki/Coding-Standards#doc-strings-and-comments